tools/dev/README.md

```diff
@@ -41,3 +41,71 @@ Sample output
     Processing whisks_design_document_for_activations_db_filters_v2.1.0.json
             - whisks-filters.v2.1.0-activations.js
     Generated view json files in /path/too/tools/build/views
+
+## IntelliJ Run Config Generator
+
+This script enables creation of [Intellij Launch Configuration][1] in _<openwhisk home>/.idea/runConfigurations_ 
+with name controller0 and invoker0. For this to work your Intellij project should be [directory based][3]. If your 
+project is file based (uses ipr files) then you can convert it to directory based via _File -> Save as Directory-Based Format_
+
+### Usage
+
+First setup OpenWhisk so that Controller and Invoker containers are up and running. Then run the script
+
+    ./gradlew -p tools/dev intellij
+    
+It would inspect the running docker containers and then generate the launch configs with name 'controller0' 
+and 'invoker0'
+
+Key points to note
+
+1. Uses ~/tmp/openwhisk/controller (or invoker) as working directory
+2. Changes the PORT to linked one. So controller gets started at 10001 only just like container
+
+Now the docker container can be stopped and application can be launched from within the IDE
+
+**Note** - Currently only controller can be run from IDE. Invoker posses some [problems][2]
```

Comment:
You use the running controller container to capture the env settings?

Reply:
Yes

Reply:
s/only controller/only the controller
and `.` at end of sentence 🙏